modules/notification/ui/ui.go

```diff
@@ -62,7 +62,7 @@ func (ns *notificationService) NotifyCreateIssueComment(doer *models.User, repo
 	_ = ns.issueQueue.Push(opts)
 }
 
-func (ns *notificationService) NotifyNewIssue(issue *models.Issue) {
+func (ns *notificationService) NotifyNewIssue(issue *models.Issue, mentions []*models.User) {
 	_ = ns.issueQueue.Push(issueNotificationOpts{
```

Comment:
Also we need send mentions notification to ui, of course that could be another PR.

Reply:
done